website/cue/reference/remap/functions/parse_nginx_log.cue

```diff
@@ -93,5 +93,33 @@ remap: functions: parse_nginx_log: {
 				host:      "localhost:8081"
```

Comment:
Let's also mention which fields might not be present when parsing a `ingress_upstreaminfo` log.